Feature/fe/506/show hide password

mtkamiloglu opened the pull request:
I added show/hide password toggle to these four pages:
* Signup page
* Login page
* Forgot password page
* Change password page

I also added some error handling to login page.

yunusemrealtug commented:
In change password, why does not old password field have this option? Also, new passworf and confirm new password depends on one variable i guess. Should not they work distinctly?

mtkamiloglu commented:
I overlooked a bug on the change password page, I fixed it. I fixed the other pages so that the passwords appear and disappear independently of each other. Can you review again @yunusemrealtug

yunusemrealtug reviewed (approved):
Works well, thanks for good work.
